webapp/components/create_comment.jsx

```diff
@@ -263,6 +263,25 @@ export default class CreateComment extends React.Component {
         $('.post-right__scroll').parent().scrollTop($('.post-right__scroll')[0].scrollHeight);
 
         this.setState({message});
+
+        this.checkMessageLength(message);
+    }
+
+    checkMessageLength(message) {
```

Comment:
Wondering if we should have this function be a common function to use here and https://github.com/mattermost/platform/pull/4867/files#diff-9c47c7c15d2564d7af567a75ba50ce9dR228